src/globus_sdk/services/search/client.py

```diff
@@ -397,6 +397,62 @@ def delete_by_query(
         log.info(f"SearchClient.delete_by_query({index_id}, ...)")
         return self.post(f"/v1/index/{index_id}/delete_by_query", data=data)
 
+    def batch_delete_by_subject(
+        self, index_id: UUIDLike, data: dict[str, t.Any]
+    ) -> response.GlobusHTTPResponse:
+        """
+        Delete data in a Search index as an asynchronous task, deleting multiple
+        documents based on their ``subject`` values.
+
+        A ``task_id`` value will be included in the response.
+
+        :param index_id: The index in which to delete data
+        :type index_id: str or UUID
+        :param data: a batch-delete-by-subject document listing documents to delete
+        :type data: dict
+
+        .. tab-set::
+
+            .. tab-item:: Example Usage
+
+                .. code-block:: python
+
+                    sc = globus_sdk.SearchClient(...)
+                    query_data = {
+                        "q": "user query",
+                        "filters": [
+                            {
+                                "type": "range",
+                                "field_name": "path.to.date",
+                                "values": [{"from": "*", "to": "2014-11-07"}],
+                            }
+                        ],
```

Comment:
Is this relevant to the usage example?

Reply:
I just was about to comment. I pushed an amend to remove this. It was a copy-pasta mistake.

Reply:
Any idea where I submit a request to have github include a pasta emoji in their list of reaction emojis?